### ISSUE TITLE
SplitButton - Narrator does not read ariaDescription

### ISSUE THREAD

**Issue by Sentri** [microsoft/fluentui#15721]
### Environment Information

- **Package version(s)**: office-ui-fabric-react 7.144.1
- **Browser and OS versions**: Microsoft Edge Version 88.0.673.0 (Official build) dev (64-bit), Windows 10 Version 1909 Build 18363.1139

### Please provide a reproduction of the bug in a codepen:

Turn on Windows Narrator. Open the following codepen and focus the split button:
https://codepen.io/savannahbourgeois/pen/RwRpvvo

#### Actual behavior:

On focus, Windows Narrator reads: "Standard; split button; collapsed".
The `ariaDescription` value is not read.

#### Expected behavior:

Narrator should also read the value of `ariaDescription`:
"Standard; split button; collapsed; Sample Extra Description"

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: High. This is an accessibility bug affected by Microsoft internal SLA.

Products/sites affected: Power Query Online, Power Apps, Power BI, Power Automate (potentially all of Office Online as well)

Additional information from Office Online team:

> In FluentUI, when the split button has an ariaDescription passed, the ariaDescription is put into a span and they set aria-describedBy to point to that span. This works in the case of most buttons, however for the split button, the aria-describedBy is getting set on the inner primary button instead of the outer div that receives focus. 

>To fix this, it should hopefully be a one-line change in Fluent. They need to add ariaDescription to buttonProps https://github.com/microsoft/fluentui/blob/0b370e43432789ee9b170f60a8276b594493ca7a/packages/react-internal/src/compat/components/Button/BaseButton.tsx#L227 which is then passed to onRenderSplitButtonContent https://github.com/microsoft/fluentui/blob/0b370e43432789ee9b170f60a8276b594493ca7a/packages/react-internal/src/compat/components/Button/BaseButton.tsx#L253

>Then in onRenderSplitButtonContent, you will need to ensure that ariaDescribedBy https://github.com/microsoft/fluentui/blob/0b370e43432789ee9b170f60a8276b594493ca7a/packages/react-internal/src/compat/components/Button/BaseButton.tsx#L602 is set properly to the rendered span for the ariaDescription. I'm not sure how mergeAriaAttributeValues works but it is possible that there will need to be an additional step since it looks like ariaDescription is getting set directly to a string in the line linked above when aria-DescribedBy takes in an ID.

**Comment by paulgildea** [comment]
@Sentri Thanks for filing this issue!

@smhigley FYI on the model

@tomi-msft Let's try to prioritize this one and coordinate with @Sentri if there is a workaround in the mean time.

@JustSlone FYI

**Comment by khmakoto** [comment]
So, I think a fix for this shouldn't be that hard to make. However, one thing I've noticed so that we unblock you in the meantime is that if you pass `aria-description` instead of `ariaDescription` it at least seems to work in the codepen you provided.
